__tests__/kata1.fizzBuzz.test.js

```diff
@@ -2,18 +2,21 @@ const { fizzBuzz } = require('../src');
 
 describe('fizzBuzz', () => {
   it('returns Fizz when passed a multiple of 3', () => {
-
+      expect(fizzBuzz(3)).toBe('Fizz')
   });
 
   it('returns Buzz when passed a multiple of 5', () => {
-
+      expect(fizzBuzz(5)).toBe('Buzz')
   });
 
   it('returns FizzBuzz when passed a multiple 3 and 5', () => {
-
+      expect(fizzBuzz(15)).toBe('FizzBuzz')
+      expect(fizzBuzz(30)).toBe('FizzBuzz')
+      expect(fizzBuzz(90)).toBe('FizzBuzz')
   });
 
   it('returns the number when it isn\'t a multiple of 3 or 5', () => {
-
+    expect(fizzBuzz(4)).toBe(4)
```

Comment:
Ensure consistency in indentation. Either go for 4 spaces or 2 spaces. I personally recommend 2.